contributing_to_docs/doc_guidelines.adoc

```diff
@@ -20,6 +20,10 @@ It is not necessary to update existing content to unwrap lines, but you can remo
 In the Atom editor, you can use `Ctrl`+`J` to undo hard wrapping on a paragraph.
 ====
 
+* The following books are auto-generated and should not be manually edited:
+** `cli_reference`
```

Comment:
I'd suggest listing them with the book/section name and then the relevant files, because they're different between the two and might not be super obvious. Maybe something like this:

```
* The following content is auto-generated each release and should not be manually edited:

** The OpenShift CLI (`oc`) command references (`modules/oc-by-example-content.adoc` and `modules/oc-adm-by-example-content.adoc`)
** The API references (anything under the `rest_api` folder that is in a `<topic>_apis` subfolder)
```

@jboxman-rh Can you confirm that this is correct? Technically there are some files under rest_api/ that can be edited I believe, so I didn't want to make a blanket statement that the entire folder was off limits.

Reply:
More or less. These as well:

- https://docs.openshift.com/container-platform/4.10/rest_api/objects/index.html
- https://docs.openshift.com/container-platform/4.10/rest_api/index.html

Thanks.